icgcget/cli.py

```diff
@@ -73,6 +75,45 @@ def logger_setup(logfile, verbose):
     return logger
 
 
+def docker_cleanup(staging):
+    try:
+        os.remove(staging + '/cidfile')
+    except OSError:
+        pass
+    env = dict(os.environ)
+    env['PATH'] = '/usr/local/bin:' + env['PATH']
+    args = ['docker', 'ps', '-a', '-q', '-f', 'status=exited']
+    exited_containers = subprocess.Popen(args, stdout=subprocess.PIPE, env=env)
+    container_ids = exited_containers.stdout.read().splitlines()
+    if container_ids:
+        print container_ids
+        args = ['docker', 'rm', '-v']
+        args.extend(container_ids)  # remove any stopped containers
+        devnull = open('/dev/null', 'w')
+        subprocess.call(args, stdout=devnull, env=env)
+
+
+def subprocess_cleanup(json_path):
+    session = load_json(json_path, False)
+    if session:
+        if session['container']:
+            print session['container']
+            args = ['docker', 'rm', '-f', session['container']]
+            subprocess.call(args)  # try to stop the last running container
+        print 'exit'
+        for pid in session['subprocess']:  # kill any existing subprocessess
+            try:
+                os.kill(pid, 0)
+            except OSError:
+                continue
+            os.kill(pid, signal.SIGKILL)
+            try:
+                os.kill(pid, 0)
+                print "Unable to kill client process with pid {}".format(pid)
```

Comment:
This is intended.  If the process is still running after sending `SIGKILL` I want to warn the user about it.  If it isn't running, there will be an error and there is no need to print the warning.